src/components/Agreements/ViewAgreement/Sections/FinancesAgreementLines.js

```diff
@@ -30,14 +31,28 @@ export default class FinancesAgreementLines extends React.Component {
     poline:   <FormattedMessage id="ui-agreements.agreementLines.poline" />,
   }
 
+  polineFormatter(line) {
+    const poLineNumber = get(line, ['po_line_number']);
+    if (poLineNumber === undefined) {
+      return '-';
+    }
+    const purchaseOrderId = get(line, ['purchase_order_id']);
```

Comment:
@julianladisch 
Can we assume we'll always have a defined `line` and a defined `purchase_order_id` and `id` on a line?

(You can always define this function as `poLineFormatter(line = {}) { .... }` to give `line` a default value and ensure it's always defined.)